src/environment/local_environment.py

```diff
@@ -8,6 +8,20 @@
 class LocalEnvironment(IEnvironment):
     def __init__(self):
         super().__init__()
-        env_vars: Dict[str, Optional[str]] = dotenv_values("../.env")
+        env_vars: Dict[str, Optional[str]] = self._load_env_vars()
         env_vars['TO_LANGUAGES'] = env_vars['TO_LANGUAGES'].split(',')
         self.set_config(config=env_vars)
+
+    @staticmethod
+    def _load_env_vars() -> Dict[str, Optional[str]]:
+        possible_paths = [
+            ".env",
+            "../.env",
+            "../../.env",
+            os.path.join(os.path.dirname(__file__), ".env"),
+            os.path.join(os.path.dirname(__file__), "../.env"),
+        ]
+        for path in possible_paths:
+            if os.path.exists(path):
+                return dotenv_values(path)
```

Comment:
Is it possible to optimise this? Is there a way we can find the .env file within the project rather than check filepaths instead?